staging/src/k8s.io/client-go/tools/events/event_broadcaster.go

```diff
@@ -102,16 +102,20 @@ func newBroadcaster(sink EventSink, sleepDuration time.Duration, eventCache map[
 	}
 }
 
+func (eventBroadcaster *eventBroadcasterImpl) Shutdown() {
+	eventBroadcaster.Broadcaster.Shutdown()
+}
+
 // refreshExistingEventSeries refresh events TTL
-func (e *eventBroadcasterImpl) refreshExistingEventSeries() {
+func (eventBroadcaster *eventBroadcasterImpl) refreshExistingEventSeries() {
```

Comment:
IIRC the convention in golang is to use a short name for receivers.

Reply:
I was trying to make kubernetes-verify job happy.